### ISSUE TITLE
How to resume pipeline at a specific point

### ISSUE THREAD

**Issue by MonicaSteffi** [nf-core/ampliseq#151]
Dear All,
I am trying nfcore pipeline for 8900 fastq files. I started and it run successfully till dada-merge step. Because of my silly mistake in making metadata file, It stopped at the step of classifier. It took nearly 14 hours to complete till this step. I got the error as "ValueError: Sample IDs found in the table are missing in the metadata: {'run1-253056XS-S107-L001'". Now I corrected metadata file. 

How do I resume this analysis? or start from the classifier step?

**Comment by erikrikarddaniel** [comment]
The standard way of resuming Nextflow workflows is to start the workflow again and pass the `-resume` flag to the command line. It then calculates which steps are already done and which are not. See the documentation: https://www.nextflow.io/docs/latest/getstarted.html#modify-and-resume.

**Comment by MonicaSteffi** [comment]
> The standard way of resuming Nextflow workflows is to start the workflow again and pass the `-resume` flag to the command line. It then calculates which steps are already done and which are not. See the documentation: https://www.nextflow.io/docs/latest/getstarted.html#modify-and-resume.

Hi thank you for the quick reply.
I checked the nextflow log details. I got the following:

2020-05-19 07:50:43	11h 45m 9s    	stoic_hilbert        	ERR   	cd23988d88 	c57af593-7a97-4fee-a3fa-1d18b2e0e825	nextflow run nf-core/ampliseq -profile standard -c nextflow_standard.config --reads data --FW_primer TCGTCGGCAGCGTCAGATGTGTATAAGAGACAGCCTACGGGNGGCWGCAG --RV_primer GTCTCGTGGGCTCGGAGATGTGTATAAGAGACAGGACTACHVGGGTATCTAATCC --metadata meta_data.tsv --multipleSequencingRuns --trunclenf 232 --trunclenr 227 --keepIntermediates --min_frequency 0 --classifier silva-132-99-nb-classifier.qza


So I run the same command with --resume c57af593-7a97-4fee-a3fa-1d18b2e0e825. But it did start from the beginning
